book/src/creating_templates.md

```diff
@@ -75,6 +75,14 @@ recognized:
   #[template(path = "hello.html", print = "all")]
   struct HelloTemplate<'a> { ... }
   ```
+* `block` (as `block = "block"`): renders a block by itself, useful for partial
```

Comment:
This would be clearer if the name of the example block were something other than "block".